test/it/src/test/java/uk/co/blackpepper/bowman/test/it/SimpleEntityIT.java

```diff
@@ -80,4 +80,21 @@ public void canPostEntityWithAssociatedProxy() {
 		
 		client.post(sent);
 	}
+	
+	@Test
+	public void canPutEntityWithAssociatedProxy() {
```

Comment:
Thanks for adding this integration test. A few thoughts:

- As written, this test isn't verifying you can PUT with an associated proxy object - a better name might be `canPutEntity`.
- I think you should re-GET the resource from the server after PUTting it before asserting on its properties. Otherwise all you're doing is verifying that the client-side object is getting updated when you call its setters.